src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Hands-On! Building Interactive Training With A-Frame
+author: twitter|salvadelapuente|Salvador de la Puente
+date: 2017-08-08 12:00:00
+layout: blog
+image:
+  src: hands-on-i.png
+---
+
+In an industry where manipulation of complex mechanisms involves tens of tools, expensive equipment and even safety risks, virtual reality helps to increase operator safety and reduce material costs. In addition, WebVR puts everything just one link away.
```

Comment:
It would be cool to provide a link to the demo. I would like personally try it first to decide if I want to read the article.